src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -340,28 +316,14 @@ export const VenueLandingPage: React.FC = () => {
                             </div>
                           )}
 
-                          {user ? (
-                            <EventPaymentButton
-                              event={venueEvent}
-                              venueId={venueId}
-                              isUserVenueOwner={!!isUserVenueOwner}
-                              selectEvent={() => setSelectedEvent(venueEvent)}
-                              setIsPaymentModalOpen={setIsPaymentModalOpen}
-                              paymentConfirmationPending={
-                                eventPaidSuccessfully === venueEvent.id
-                              }
-                            />
-                          ) : (
-                            <button
-                              className="btn btn-primary buy-tickets-button"
-                              onClick={() => {
-                                setSelectedEvent(venueEvent);
-                                openAuthenticationModal();
-                              }}
-                            >
-                              Buy tickets
-                            </button>
-                          )}
+                          <button
+                            className="btn btn-primary buy-tickets-button"
```

Comment:
This still mentions the class `buy-tickets-button`, so presumably needs to have some styling/similar cleaned up still, and/or this button/etc removed from here entirely. You'd need to look more into the logic of this page to make that decision properly.

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40